web/src/components/forms/access-rule/CreateForm.tsx

```diff
@@ -45,7 +45,7 @@ export const accessRuleFormDataTargetToApi = (
   }
   for (const k in target.multiSelects) {
     t.with[k] = {
-      groupings: target.argumentGroups[k] || {},
+      groupings: target.argumentGroups?.[k] || {},
```

Comment:
Surprising this didn't get picked up in any of our automated or UAT testing.
I will add a ticket in the backlog to track adding coverage to tests

Reply:
There is obviously a typing challenge here between the form types and the api types.
Ideally we would have picked up this issue as a type error